frappe/patches/v12_0/setup_log_settings.py

```diff
@@ -0,0 +1,10 @@
+import frappe
```

Comment:
Remove this patch. Let users voluntarily create these settings. There might be some user-created custom doctype which ends with "Log" and they might not want to delete data older than 30 days. Also, data of doctype like "Energy Point Log" is used to get aggregated points of the user, so clearing them does not make sense either. 

**Note:** You can create settings for doctypes which already had code to clear 30days or older records (like "Actvity Log" and "Error Log")